posts/turfs-formerly-gangzones-module.mdx

```diff
@@ -0,0 +1,40 @@
+export const meta = {
+  date: '2019-10-19T04:20:00',
+  author: 'J0sh'
+};
+
+# Turfs (formerly gangzones) module
```

Comment:
Probably more of a question for Josh - but should this be just "Turf", plural being "Some turf"

See also: Lego/some lego/a piece of lego vs "Legos"

Reply:
Probably should be Turf, yeah.

Also, this whole code-review process is going to make putting out content very easy and transparent! We can catch typos and stuff like this early then once it's live, copy it out to facebook/burgershot/twitter/etc